exporter/collector/config.go

```diff
@@ -49,6 +49,9 @@ type MetricConfig struct {
 	SkipCreateMetricDescriptor bool   `mapstructure:"skip_create_descriptor"`
 	// If a metric belongs to one of these domains it does not get a prefix.
 	KnownDomains []string `mapstructure:"known_domains"`
+
+	// If true, set the instrumentation_source and instrumentation_version labels
+	InstrumentationLibraryLabels bool `mapstructure:"instrumentation_library_labels"`
```

Comment:
would be good to mention the default is true

Reply:
Good idea. Done.